lua/telescope/_extensions/ui-select.lua

```diff
@@ -29,7 +29,7 @@ return require("telescope").register_extension {
             }
             for idx, item in ipairs(items) do
               local client_id, title
-              if vim.version and vim.version.cmp(vim.version(), vim.version.parse "0.10-dev") >= 0 then
+              if vim.version and vim.version.cmp(vim.version(), vim.version.parse "0.10.1-dev") >= 0 then
```

Comment:
If I understood it correctly we don't need to use parse here we can use something like this:

`vim.version.cmp(vim.version(), { 0, 10, 0 })`

Also it maybe good to remove `-dev` meta, because it does not matter anyways, as stated here [docs ref](https://neovim.io/doc/user/lua.html#version-range) :

![image](https://github.com/nvim-telescope/telescope-ui-select.nvim/assets/7147208/b92f4071-df14-4c00-a72b-06095d5fe477)

Reply:
I just tested it, and it does not error out when removing `-dev` while comparing "0.10.0", so it appears it's there for a reason.  That also rules out using a tuple instead of version parsing the string. I will leave it to the maintainers to take action on that, since I don't know their reasoning.

And thank you, that is rewarding to hear :-)

Reply:
Can you merge this? 🙏🏽

Reply:
@camiloaromero23, I don't have write access. I will try to ping the maintainers.